README.md

```diff
@@ -1 +1,32 @@
 # LoggingFormats.jl
+
+This package is an aggregation of various useful functions to use used in 
+[FormatLogger](https://github.com/JuliaLogging/LoggingExtras.jl#formatlogger-sink) from package 
+[LoggingExtras](https://github.com/JuliaLogging/LoggingExtras.jl).
+
+Currently, there are following functions available:
+- `make_log_truncated`
+
+## Truncate long variables and messages
+
+`make_log_truncated(max_var_len=5_000)` is a function which formats data in similar manner as `ConsoleLogger`, 
```

Comment:
Perhaps clarify that the truncation is applied to i) the log message and ii) any extra added variables individually (the full log output is not truncated, that could be an extension for later).

Reply:
Readme clarified